File: Backend/Sistema de citas/Sistema de citas/Model/Quotes.cs
using System.ComponentModel.DataAnnotations;

namespace Sistema_de_citas.Model
{
    public class Quotes
    {
        [Key]
        public int Id { get; set; }
        public string title { get; set; } 
        public string quote_description { get; set; }
        public DateTime quote_date { get; set; }
        public string quote_state { get; set; }
        public int UsersId { get; set; }
        public int ServiciosId { get; set; }
        public Users Users { get; set; }
        public Servicios Servicios { get; set; }
    }
}

File: Frontend/Sistema_De_Citas/src/app/models/Quotes.ts
export class Quotes {
    id!: number;
    title!: string;
    quote_description!: string;
    quote_date!: Date;
    quote_state!: string;
    usersId!: number;
    serviciosId!: number;

    constructor(
        id: number,
        title: string,
        quote_description: string,
        quote_date: Date,
        quote_state: string,
        usersId: number,
        serviciosId: number
    ) {
        this.id = id;
        this.title = title;
        this.quote_description = quote_description;
        this.quote_date = quote_date;
        this.quote_state = quote_state;
        this.usersId = usersId;
        this.serviciosId = serviciosId;
    }
}